ui/src/app/index/login.component.ts

```diff
@@ -30,6 +30,7 @@ export class LoginComponent implements ViewWillEnter, AfterContentChecked, OnDes
   protected formIsDisabled: boolean = false;
   protected popoverActive: "android" | "ios" | null = null;
   protected showPassword: boolean = false;
+  protected showPasswordBackend: boolean = false;
```

Comment:
why creating seperate variable for this?

Reply:
I'm no sure why `showPassword` is here. This variable is from commit. I think this variable is not used.
Can I used this variable?
https://github.com/OpenEMS/openems/pull/2969/commits/047ed2cc965652ff11e891832ff51989dcc49c16

Reply:
I understood this problem. Our original diff is use `showPassword` and it forked old release(there is no using `showPassword` when this old release).
so when making this PR from our diffs, AI shift our function variable to `backendShowPassword`...